configs/securitygroups.yml

```diff
@@ -345,3 +345,21 @@ VCSSync:
         - include: global-ping
     outbound:
         - include: global-any
+
+signig-worker:
+    description: security group for signing workers
+    regions:
+        us-west-1: vpc-7a7dd613
+        us-west-2: vpc-cd63f2a4
+        us-east-1: vpc-b42100df
+    inbound:
+        # traffic from other masters
+        - proto: tcp
+          ports:
+            - 22  # ssh
+        - include: admin-access
+        - include: observium
+        - include: infra-puppetize
+        - include: global-ping
+    outbound:
+        - include: global-any
```

Comment:
Oh, and don't forget to add flows for these new subnets to the scl3 firewall, as necessary, and to add firewall tests for the new flows.  Callek can help you with that.